src/test/resources/esapi/ESAPI.properties

```diff
@@ -498,7 +498,7 @@ Validator.Redirect=^\\/test.*$
 Validator.HTTPScheme=^(http|https)$
 Validator.HTTPServerName=^[a-zA-Z0-9_.\\-]*$
 Validator.HTTPCookieName=^[a-zA-Z0-9\\-_]{1,32}$
-Validator.HTTPCookieValue=^[a-zA-Z0-9\\-\\/+=_ ]*$
+Validator.HTTPCookieValue=^[a-zA-Z0-9\\-\\/+=_ ]{1,1024}$
```

Comment:
Similar comment: I thought you had planed to change these regexes to a range of `{0,1024}` to allow for cookies with no value? Did that not work?

Ideally we should make these changes too ALL the ESAPI properties files under src/test/resources so they don't diverge over time.

Reply:
see commit [5246396](https://github.com/ESAPI/esapi-java-legacy/pull/663/commits/5246396ee4b7a2aa77c4f1fa20375721e57ee02d)

Reply:
Okay, but can you also change this to be like the others in **ESAPI.properties**?
```
src/test/resources/esapi/ESAPI-CommaValidatorFileChecker.properties:471:Validator.HTTPCookieValue=^[a-zA-Z0-9\\-\\/+=_ ]*$
src/test/resources/esapi/ESAPI-DualValidatorFileChecker.properties:472:Validator.HTTPCookieValue=^[a-zA-Z0-9\\-\\/+=_ ]*$
src/test/resources/esapi/ESAPI-QuotedValidatorFileChecker.properties:470:Validator.HTTPCookieValue=^[a-zA-Z0-9\\-\\/+=_ ]*$
src/test/resources/esapi/ESAPI-SingleValidatorFileChecker.properties:470:Validator.HTTPCookieValue=^[a-zA-Z0-9\\-\\/+=_ ]*$
```
I'd prefer to keep them as similar as possible since when then diverge a lot there is a lot of diffing that needs to be done to get them sync'd back up. Otherwise, we should strip down those other for properties files to them minimal set used to make the tests succeed.

Reply:
Maybe I messed this up, but I fixed it in src/main before src/test.  (Just what I had open.)

I’ll double check when I get upstairs

Reply:
It used to be src/test and src/main but I thought src/main was replaced with configuration/

Reply:
No, you didn't mess it up. I just asked that you make the same change to the other 4 properties files in 'src/test/resources'.
```
src/test/resources/esapi/ESAPI-CommaValidatorFileChecker.properties:471:Validator.HTTPCookieValue=^[a-zA-Z0-9\\-\\/+=_ ]*$
src/test/resources/esapi/ESAPI-DualValidatorFileChecker.properties:472:Validator.HTTPCookieValue=^[a-zA-Z0-9\\-\\/+=_ ]*$
src/test/resources/esapi/ESAPI-QuotedValidatorFileChecker.properties:470:Validator.HTTPCookieValue=^[a-zA-Z0-9\\-\\/+=_ ]*$
src/test/resources/esapi/ESAPI-SingleValidatorFileChecker.properties:470:Validator.HTTPCookieValue=^[a-zA-Z0-9\\-\\/+=_ ]*$
```

Everyone forgets to update them. Including myself. The need to have the regex changed so that the range check `{0,1024}` is added at the end instead of the '*' at the end.